keyboards/z12/keymaps/zigotica/oled.c

```diff
@@ -17,23 +17,36 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "zigotica.h"
 
 static void render_status(void) {
-    oled_write_P(PSTR("z12 v1.0\n"), false);
-    oled_write_P(PSTR("Layer: "), false);
     switch (get_highest_layer(layer_state)) {
-        case _VIM:
-            oled_write_P(PSTR("VIM      \n\nBUFFER         SCROLL"), false);
+        case _BASE:
+            oled_write_P(PSTR("Volume         Scroll"), false);
+            oled_write_P(PSTR("                     "), false);
```

Comment:
Could add advance page here, instead of the empty lines. Would be smaller, firmware wise.  Though I can understand why this may be preferred.

Reply:
Hi @drashna thanks a lot for the suggestion. I would prefer to use the current code, might not be so optimized, but it's easier to see and understand what will be visible on screen for each layer.

Reply:
I figured as much.